tripal_chado/includes/tripal_chado.fields.inc

```diff
@@ -208,6 +212,9 @@ function tripal_chado_bundle_fields_info_base(&$info, $details, $entity_type, $b
       $base_info['type'] = 'text';
       $base_info['settings']['text_processing'] = 0;
     }
+
+    dpm($base_info);
```

Comment:
How did this get added? Was testing and saw dpms when finding new fields 🤣 

```suggestion
```

Reply:
Confirmed this is the only one that was introduced.

Reply:
ack!  I also have no idea how this got added, im so sorry.  Thank you for confirming this was the only one.